docs/release/releases/release-1.12.md

```diff
@@ -0,0 +1,37 @@
+# Cluster API v1.12
+
+## Timeline
+
+The following table shows the preliminary dates for the `v1.12` release cycle.
+
+| **What**                                             | **Who**      | **When**                    | **Week** |
+|------------------------------------------------------|--------------|-----------------------------|----------|
+| Start of Release Cycle                               | Release Lead | Monday 18th August 2025     | week 1   |
+| Schedule finalized                                   | Release Lead | Friday 22nd August 2025     | week 1   |
+| Team finalized                                       | Release Lead | Friday 22nd August 2025     | week 1   |
```

Comment:
```suggestion
| Start of Release Cycle                               | Release Lead | Monday 25th August 2025     | week 1   |
| Schedule finalized                                   | Release Lead | Friday 29nd August 2025     | week 1   |
| Team finalized                                       | Release Lead | Friday 29nd August 2025     | week 1   |
```

Let's move the start of the cycle to next week (I think it is more correct, considering GA was this week)

Reply:
Note: this will require to change week numbers in all the following lines 😅